pkg/configmap/configmap.go

```diff
@@ -44,16 +46,59 @@ func BuildConfigMap(namespace string, kueueCfg kueue.KueueConfiguration) (*corev
 	return cfgMap, nil
 }
 
-func defaultKueueConfigurationTemplate(kueueCfg kueue.KueueConfiguration) *configapi.Configuration {
-	managedJobsConversion := ptr.Deref(kueueCfg.ManageJobsWithoutQueueName, kueue.QueueName)
-	managedJobsBool := false
-	if managedJobsConversion == kueue.QueueName {
-		managedJobsBool = false
-	} else if managedJobsConversion == kueue.NoQueueName {
-		managedJobsBool = true
-	} else {
-		managedJobsBool = false
+func mapOperatorIntegrationsToKueue(integrations *kueue.Integrations) *configapi.Integrations {
+
+	return &configapi.Integrations{
+		Frameworks:         buildFrameworkList(integrations.Frameworks),
+		ExternalFrameworks: buildExternalFrameworkList(integrations.ExternalFrameworks),
+		LabelKeysToCopy:    buildLabelKeysCopy(integrations.LabelKeysToCopy),
+	}
+}
+
+func buildFrameworkList(kueuelist []kueue.KueueIntegration) []string {
+	// Upstream kueue uses lowercase names for these.
+	// This does not fit our api review so we are converted before building it into
+	// the configmap.
+	conversionMap := map[string]string{}
```

Comment:
@kannon92, can we have a detailed discussion around this? It might be helpful to address the opt-in namespaces feature, which also uses some of these bits.
xref: https://github.com/openshift/kueue-operator/pull/87/files#diff-5194bfa0ec2a70b8e809d8cb8064d299c1db55f0e154857a6d27eada23bfafa3R30